Assignment2/Q1/src/NodeBinaryTree.java

```diff
@@ -0,0 +1,93 @@
+import javax.xml.soap.Node;
+
+/**
+ * Created by Alice Colt.
+ *
+ * Implementation of Node for BinaryTree with constructor, getters, setters and compareTo
+ *
+ */
+public class NodeBinaryTree
+{
+    // Value of node
+    private int value;
+
+    // References to root, left and right
+
+    private NodeBinaryTree root;
```

Comment:
"parent" would be a much better name than "root". "root" implies the furthest ancestor with no parents. "parent" implies direct ancestor, which is what this is.